### PR TITLE
SISRP-14135 - Front End Dev: View-As Card with Names for Advising Pt 1

### PR DESCRIPTION
**Pull request by winstonkim**
https://jira.berkeley.edu/browse/SISRP-14135

### CODE REVIEW

File: src/assets/templates/widgets/toolbox/view_as.html [hunk 1]
```diff
@@ -57,7 +60,11 @@
                     <button type="submit" class="cc-button-link" data-ng-click="admin.updateIDField(user.ldap_uid)" data-ng-bind="user.ldap_uid"></button>
                   </td>
                   <td class="cc-table-top-border">
-                    <span data-ng-bind="user.student_id || 'none'">
+                    <span data-ng-bind="user.student_id || 'none'"></span>
+                  </td>
+                  <td class="cc-table-top-border cc-table-ellipsis">
+                    <span data-ng-if="user.first_name && user.last_name" data-ng-bind="user.first_name + ' ' + user.last_name"></span>
+                    <span data-ng-if="!user.first_name && !user.last_name">none</span>
```
Comment (christianvuerings) [inline src/assets/templates/widgets/toolbox/view_as.html]: @winstonkim should this be a `||` instead?
Not sure whether this is possible but what will happen if the user only has a `first_name` or `last_name`?

**Comment by christianvuerings** [comment]
lgtm after comment is addressed

**Comment by winstonkim** [comment]
@christianvuerings made the changes you requested. take a look.

**Comment by christianvuerings** [comment]
@winstonkim awesome. Looks like a small change in the code but will be such a big improvement for the advisor / admin user!

**Pull request merged**